### ISSUE TITLE
[Explore]: Source code to Image conversion

### ISSUE THREAD

**Issue by akrishna90** [cloudfoundry/cf-k8s-controllers#185]
### Background

**Blocked By** 
[[Explore]: Packaging source code as Carvel's imgpkg ](https://github.com/cloudfoundry/cf-k8s-api/issues/35)

Based on the recommendation from cloudfoundry/cf-k8s-api#35, explore on how to convert source code into the recommended image format.

**Things to remember**

- Ease of Scalability
- Performance

### Acceptance Criteria

The pair working on this story can decide b/w documenting the decision in an ADR or a proposal document. 

### Dev Notes

- One approach is to implement this as a separate service 'registry-service' or another way would be to make it part of the api-shim itself.

**Comment by tcdowney** [comment]
> Superseded by [cloudfoundry/cf-k8s-api#46](https://github.com/cloudfoundry/cf-k8s-api/issues/46)

Closing this ^^
